examples/djangoproject/settings.py

```diff
@@ -39,7 +39,8 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        # configure root templates
+        'DIRS': [os.path.join(BASE_DIR, "templates")],
```

Comment:
this is also unnecessary

Reply:
This is necessary :imp: 
Unlike you I test every case...

Reply:
> This is necessary 👿

Actually no. Check how django registers app. This is actually not necessary